iceoryx_utils/include/iceoryx_utils/internal/concurrent/lockfree_queue/cyclic_index.inl

```diff
@@ -82,4 +82,4 @@ bool CyclicIndex<CycleLength, ValueType>::isOneCycleBehind(const CyclicIndex& ot
     return (thisCycle + 1 == otherCycle);
 }
 } // namespace concurrent
-}
\ No newline at end of file
+} // namespace iox
```

Comment:
New line missing?

Reply:
We do not need this since C++11. It is a requirement from the old C world which was abandoned with C++11.

Reply:
See comment above